### PR TITLE
Update import workflow (CLA passing)

### PR DESCRIPTION
**Pull request by belluzj**
* Simplify the workflow
* Make sure the CLA passes by using my email in commits and not opening a PR automatically

### CODE REVIEW

File: .github/workflows/import.yaml [hunk 1]
```diff
@@ -10,71 +10,55 @@ on:
...
+          git fetch origin "$SOURCE_BRANCH"
+          git branch "$SOURCE_BRANCH" "origin/$SOURCE_BRANCH"
+          git worktree add --quiet ../source "$SOURCE_BRANCH"
+
+          # Try to fetch, OK if it doesn't exist (we'll create it)
+          if git fetch origin "$TARGET_BRANCH"; then
+            echo "The target branch '$TARGET_BRANCH' already exists, updating."
+            git worktree add --quiet ../target "$TARGET_BRANCH"
+          else
+            git fetch origin main # because shallow
```
Comment (RickyDaMa) [inline .github/workflows/import.yaml:38]: I would log here that we're aware this is a new import

File: .github/workflows/import.yaml [hunk 2]
```diff
@@ -10,71 +10,55 @@ on:
...
-          git branch "$STAGING_BRANCH" "origin/$STAGING_BRANCH"
-          git worktree add --quiet ../staging "$STAGING_BRANCH"
+
+          # The clone is shallow, need to fetch other branches
+          git fetch origin "$SOURCE_BRANCH"
+          git branch "$SOURCE_BRANCH" "origin/$SOURCE_BRANCH"
+          git worktree add --quiet ../source "$SOURCE_BRANCH"
+
+          # Try to fetch, OK if it doesn't exist (we'll create it)
+          if git fetch origin "$TARGET_BRANCH"; then
```
Comment (RickyDaMa) [inline .github/workflows/import.yaml]: I'd suppress the output of this command

File: .github/workflows/import.yaml [hunk 3]
```diff
@@ -10,71 +10,55 @@ on:
...
+          SOURCE_BRANCH: ${{ github.event.inputs.branch }}
+          TARGET_BRANCH: ${{ format('import-{0}', github.event.inputs.branch) }}
         run: |
-          cd "$GITHUB_WORKSPACE/main"
+          # Create the import commit in the "target" worktree
+          cd "$GITHUB_WORKSPACE/target"
+          # Update existing branch or create a new one if it doesn't exist
           git config user.name "Import Script"
-          git config user.email "sd@daltonmaag.com"
+          git config user.email "jany.belluz@daltonmaag.com"
```
Comment (RickyDaMa) [inline .github/workflows/import.yaml]: Maybe we should make this whoever runs the workflow. Can either ask for it to be entered, or work it out automagically if GitHub has a way for you to do so

File: .github/workflows/import.yaml [hunk 4]
```diff
@@ -10,71 +10,55 @@ on:
...
           git add sources
-          git commit --quiet -m "Update from $STAGING_BRANCH
+          git status | head -n 20
```
Comment (RickyDaMa) [inline .github/workflows/import.yaml]: Why do we want this? It's just more noise in the logs - you can see the files that have changed in the PR/branch

**Review by RickyDaMa — CHANGES_REQUESTED** [review]
Largely making a to-do list for myself with this review

The README.md in the actions folder will also need to updated I think

Commented out stuff should just be deleted

File: .github/workflows/import.yaml (hunk 4 above)
Comment (belluzj) [reply .github/workflows/import.yaml]: Yes please remove, it's like a console.log for myself. Thanks

**Review by RickyDaMa — APPROVED** [review]
Now prompts for user's emails, and all other minor todos sorted :)

**Comment by MariannaPaszkowska** [comment]
@belluzj @RickyDaMa thank you!

**Pull request merged**